MAUI/DataGrid/conditional-styling.md

```diff
@@ -164,5 +164,218 @@ public class ColorConverter : IValueConverter
 
 ![Conditional column styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-columnstyle-conditional.png)
 
+## Conditional cell style
+The grid cell can be customized conditionally by writing the style with a converter for the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) control. Its BindingContext is the underlying datasource object.
 
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter"/>
+        <Style TargetType="syncfusion:DataGridCell">
+            <Setter Property="Background" Value="{Binding OrderID, Converter={StaticResource converter}}"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
+{% endhighlight %}
+{% highlight xaml tabtitle="ColorConverter.cs"%}
+public class ColorConverter : IValueConverter
+{
+    object IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo info)
+    {
+        if ((int)value < 1006)
+            return Colors.LightBlue;
+        else
+            return Colors.White;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional cell styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridcellstyle.png)
+
+## Style a cell based on MappingName and RowColumnIndex
+Styling can be applied to a particular cell based on RowIndex, ColumnIndex and MappingName property by writing the style for the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) TargetType.
```

Comment:
Styling can be applied to a particular cell based on RowIndex, ColumnIndex and MappingName property of the column by writing the style for the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) TargetType.

Reply:
Hi @NeelakandanKannan , I have addressed your feedback